src/app/selectors.ts

```diff
@@ -119,6 +119,8 @@ export const fiatConnectCashInEnabledSelector = (state: RootState) =>
   state.app.fiatConnectCashInEnabled
 export const fiatConnectCashOutEnabledSelector = (state: RootState) =>
   state.app.fiatConnectCashOutEnabled
+export const fiatConnectAccountNumberCountryOverridesSelector = (state: RootState) =>
+  state.app.fiatConnectAccountNumberCountryOverrides
```

Comment:
I believe all remote config flags currently go in app. I am not sure how easy it is to get it into state.fiatConnect since the `UPDATE_REMOTE_CONFIG_VALUES` action is under app.

Reply:
oh cool, this should be easier to do than I thought 😄